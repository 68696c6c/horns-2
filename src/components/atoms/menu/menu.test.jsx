/* global describe, it, expect */
import React from 'react'

import { render } from 'setup-test'

import Menu from '.'

describe('Menu', () => {
  it('should render as default', () => {
    const { container } = render(
      <Menu>
        <li>one</li>
        <li>two</li>
        <li>three</li>
      </Menu>
    )
    expect(container).toMatchSnapshot()
  })
  it('should render open', () => {
    const { container } = render(
      <Menu open>
        <li>one</li>
        <li>two</li>
        <li>three</li>
      </Menu>
    )
    expect(container).toMatchSnapshot()
  })
})
