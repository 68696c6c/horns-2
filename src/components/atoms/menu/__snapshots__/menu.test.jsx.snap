// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu should render as default 1`] = `
.emotion-0 {
  display: none;
  position: absolute;
  top: 0;
  z-index: 99999999999;
  box-sizing: border-box;
  min-width: 100%;
}

<div>
  <nav
    class="emotion-0"
    direction="bottom"
  >
    <li>
      one
    </li>
    <li>
      two
    </li>
    <li>
      three
    </li>
  </nav>
</div>
`;

exports[`Menu should render open 1`] = `
.emotion-0 {
  position: absolute;
  top: 0;
  z-index: 99999999999;
  box-sizing: border-box;
  min-width: 100%;
}

<div>
  <nav
    class="emotion-0"
    direction="bottom"
    open=""
  >
    <li>
      one
    </li>
    <li>
      two
    </li>
    <li>
      three
    </li>
  </nav>
</div>
`;
